docs/docs/guides/build/partitions-and-backfills/partitioning-ops.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Partitioning ops | Dagster
+description: Partitioned ops enable launching backfills, where each partition processes a subset of data.
+---
+
+# Partitioning ops
+
+:::note
+
+This page is specific to ops. To learn about partitioning assets, see "[Partitioning assets](partitioning-assets)".
+
+:::
+
+When defining a job that uses [ops](/guides/build/ops/) you can partition it by supplying <PyObject section="partitions" module="dagster" object="PartitionedConfig" /> object as its config.
+
+In this guide, we'll demonstrate to use partitions with ops and [jobs](/guides/build/jobs/).
+
+## Prerequisites
```

Comment:
nit: Do we use these "Prerequisites" sections outside of tutorials/examples anywhere else?

Reply:
We do, but not in this exact format -- this is a holdover from the legacy docs. I didn't clean up the formatting much on these pages because I don't have time, unfortunately.